alacritty_terminal/src/term/mod.rs

```diff
@@ -760,6 +763,12 @@ pub struct Term<T> {
 
     /// Terminal focus
     pub is_focused: bool,
+
+    /// Title of the window
+    title: String,
+
+    /// The stack of titles; the most recent is last
+    title_stack: Vec<String>,
```

Comment:
I almost hate to say it (I generally don't like long names), but would `saved_title_stack` or `pushed_title_stack` make things a bit better? Otherwise it may be a little bit less clear why we have a `title` and a `title_stack`. Otherwise I'd just expand the comment a tiny bit to say something about how popping a title from this stack effects the `title`.

Reply:
I honestly don't feel like `saved_` or `pushed_` improve things much, but I do see the issue itself.

I think with proper documentation on the field, this should be fine though. But I'd steer away from language like `the most recent is last`, since it might also mean that the current title is the last element. So we definitely need to be clear here.

Reply:
I updated the comments to be a bit more explanatory and clear.